src/main/java/io/r2dbc/postgresql/codec/Line.java

```diff
@@ -0,0 +1,64 @@
+package io.r2dbc.postgresql.codec;
+
+import java.util.Objects;
+
+/**
+ * Value object that maps to the {@code line} datatype in Postgres.
+ * <p>
+ * Uses {@code double} to represent the coordinates.
+ */
+public final class Line {
+
+    private final double a;
+
+    private final double b;
+
+    private final double c;
+
+    private Line(double a, double b, double c) {
+        this.a = a;
+        this.b = b;
+        this.c = c;
+    }
+
+    public static Line of(double a, double b, double c) {
```

Comment:
Would it make sense to add factory methods accepting two points or even x1/y1, x2/y2?